### PR TITLE
docs(notification): page content

### PR DESCRIPTION
**Pull request by juliengleyse**

### CODE REVIEW

File: demo/react/doc/product/components/notification.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+```javascript import
+```
+
+# Notification
+
+**Notifications provide informations at the bottom of the screen.**
```
Comment (FlorentDinet) [inline demo/react/doc/product/components/notification.mdx]: information~s~

Comment (FlorentDinet) [reply demo/react/doc/product/components/notification.mdx]: bottom of the ~screen~ user's window

File: demo/react/doc/product/components/notification.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    <>
+        Un boutton M qui declenche un toast tiny default ici
+    </>
+);
+```
+
+
+## Sizes
+
+Snackbars comes in three sizez: _tiny_, _regular_ or _big_.
```
Comment (FlorentDinet) [inline demo/react/doc/product/components/notification.mdx]: ~Snackbars~Notifications

File: demo/react/doc/product/components/notification.mdx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+() => (
+    <>
+        <img width="290" src="demo/react/doc/product/components/notification-tiny-error.png" />
+    </>
+);
+```
+
+#### Variant
+
+Tiny snackbars can display one button.
```
Comment (FlorentDinet) [inline demo/react/doc/product/components/notification.mdx]: ~snackbars~notifications

**Comment by juliengleyse** [comment]
@FlorentDinet merci

**Pull request closed without merging**